mkdocs.yml

```diff
@@ -90,7 +90,6 @@ nav:
       - 'dcrd and dcrwallet CLI Arguments': 'wallets/cli/dcrd-and-dcrwallet-cli-arguments.md'
       - 'dcrctl RPC Commands': 'wallets/cli/dcrctl-rpc-commands.md'
     - 'SPV': 'wallets/spv.md'
-    - 'Lightning Network (LN)': 'wallets/lightning.md'
```

Comment:
When we move a page, we need to add an nginx redirection here: https://github.com/decred/dcrdocs/blob/master/conf/nginx.conf

```
rewrite ^/wallets/lightning/?$               $scheme://$http_host/lightning-network/overview permanent;
```